package.json

```diff
@@ -3,6 +3,19 @@
   "version": "0.3.2",
   "description": "The Jake and trOY devopsy utility",
   "dependencies": {
```

Comment:
Looks like it added a lot of dependencies/sub-dependencies to the overall project.

Reply:
Most deps right now are for static site building as it's based on my gulpfile of the last few years. My separate static site generator project, back-burnered uses far less deps and I would expect to replace Joy code with that in a future update.